tests/suites/charmhub/download.sh

```diff
@@ -6,7 +6,7 @@ run_charmhub_download() {
 
 	ensure "test-${name}" "${file}"
 
-	output=$(juju download mysql --series xenial --filepath="${TEST_DIR}/mysql.charm" 2>&1 || true)
+	output=$(juju download mysql --series jammy --channel 8.0/stable --filepath="${TEST_DIR}/mysql.charm" 2>&1 || true)
```

Comment:
This is a bug. 3.x is successful with 
```
$ juju download mysql
Fetching charm "mysql" revision 151 using "stable" channel and base "amd64/ubuntu/22.04"
Install the "mysql" charm with:
    juju deploy ./mysql_r151.charm
```
However 2.9 cannot
```
$ juju download mysql
ERROR "mysql" has no releases in channel "stable". Type
    juju info mysql
for a list of supported channels.
```

It's related to default channel.